tests/unit/createActivities.unitTest.ts

```diff
@@ -340,6 +340,30 @@ describe("createActivity", () => {
         });
     });
 
+    context("when the payload for 'visit' activityType has startTime and endTime", () => {
+        const payload: any = {
+            activityType: "visit",
+            testStationName: "Rowe, Wunsch and Wisoky",
```

Comment:
Would be good to have some tests which actually check if the start and end times are being set correctly - At the moment, it would "succeed" regardless of correct function, due to setting of defaults. would need to be specifying something other than 'now'.

Reply:
Tried with various options, but its tough or needs more effort to investigate on sinon framework which could be used here to verify if the requestBody being sent in Put request is as expected. Will park this for now as 5819 ticket needs to be resolved to unblock 179.